### PR TITLE
Add iCloud backup

### PR DESCRIPTION
**Pull request by IDima1**

### CODE REVIEW

**Review by ivanlele — APPROVED** [review]

File: Rarime/Resources/Localizable.xcstrings [hunk 1]
```diff
@@ -344,6 +344,18 @@
     },
     "Azerbaijan" : {
 
+    },
+    "Back up manually" : {
```
Comment (ardier16) [inline Rarime/Resources/Localizable.xcstrings:348]: "Back up using a key"

File: Rarime/Resources/Localizable.xcstrings [hunk 2]
```diff
@@ -2715,7 +2742,7 @@
     "User creation failed" : {
 
     },
-    "Uses pregenerated keys" : {
+    "Uses pregenerated keys or iClould" : {
```
Comment (ardier16) [inline Rarime/Resources/Localizable.xcstrings:2745]: leave it as it was

File: Rarime/Resources/Localizable.xcstrings [hunk 3]
```diff
@@ -2838,6 +2865,9 @@
     },
     "YOU" : {
 
+    },
+    "You can restore your account using your iCloud backup or private key" : {
```
Comment (ardier16) [inline Rarime/Resources/Localizable.xcstrings:2869]: You can restore account using your iCloud backup or Private Key

File: Rarime/Resources/Localizable.xcstrings [hunk 4]
```diff
@@ -2179,6 +2197,15 @@
     },
     "Reserving..." : {
 
+    },
+    "Restore manually" : {
```
Comment (ardier16) [inline Rarime/Resources/Localizable.xcstrings:2201]: "Restore using a key"

**Review by ardier16 — APPROVED** [review]

**Pull request merged**